tests/system/tests/test_chage.py

```diff
@@ -0,0 +1,244 @@
+"""
+Test chage
```

Comment:
I didn't know about the chage command. I thought it was a typo for the word change. **Test chage (Change Age)** would be nice for anybody who may not be familiar with the shadow utilities.

Reply:
I disagree with this.  However, we could say `chage(1)`, to clarify that we refer to the program.